src/nuget/Microsoft.DotNet.Build.Tasks.Feed.nuspec

```diff
@@ -20,11 +20,11 @@
     </dependencies>
   </metadata>
   <files>
-    <file src="Microsoft.DotNet.Build.Tasks.Feed\Microsoft.DotNet.Build.Tasks.Feed.dll" target="lib\netstandard1.5" />
-    <file src="Microsoft.DotNet.Build.Tasks.Feed.net45\*.dll" target="lib\netstandard1.5" Exclude="Microsoft.DotNet.Build.Tasks.Feed\Microsoft.DotNet.Build.Tasks.Feed.dll"/>
-    <file src="Microsoft.DotNet.Build.Tasks.Feed.net45\*.dll" target="lib\net46" />
-    <file src="Microsoft.DotNet.Build.Tasks.Feed\PackageFiles\**\*.*" target="lib\netstandard1.5" />
-    <file src="Microsoft.DotNet.Build.Tasks.Feed\PackageFiles\**\*.*" target="lib\net46" />
+    <file src="Microsoft.DotNet.Build.Tasks.Feed\Microsoft.DotNet.Build.Tasks.Feed.dll" target="build\netstandard1.5" />
+    <file src="Microsoft.DotNet.Build.Tasks.Feed.net45\*.dll" target="build\netstandard1.5" Exclude="Microsoft.DotNet.Build.Tasks.Feed\Microsoft.DotNet.Build.Tasks.Feed.dll"/>
```

Comment:
Why are you putting the net45 files into the netstandard1.5 package folder?

Reply:
copying the dependencies of the feed.dll

Reply:
Wes' point is that those would be the wrong dependencies though.

Reply:
Ah will fix.

Reply:
The bin folder for the netstandard version doesn't have any deps in it, I think this is how packaging has worked. Will test before confirming.

Reply:
was missing CopyNugetImplementaions